src/index.js

```diff
@@ -30,15 +30,26 @@ const createAddresses = require("./blockchain/createAddresses")(
 );
 
 bot.onEvent(async context => {
+  // Public / Private channels
   if (context.event.isChannelsMessage || context.event.isGroupsMessage) {
+    // Unless @cooldoge is mentioned, don't react
+    if (process.env.BOT_USER_ID) {
+      const re = new RegExp(`<@${process.env.BOT_USER_ID}>`);
+      if (!re.test(context.event.text)) {
+        return;
+      }
+    }
+
+    // console.log(context);
```

Comment:
Should we remove it? I'd start to have higher code standards for the bot 🐶

Reply:
hahaha